### PR TITLE
Publicity column for own documents [#183999201]

### PR DESCRIPTION
**Pull request by rpulkka**
https://183999201.dev.laji.fi/vihko/ownSubmissions
https://www.pivotaltracker.com/n/projects/2346653/stories/183999201

Now there's a column that sorts own submissions by publicity.

### CODE REVIEW

File: projects/laji/src/app/shared-modules/own-submissions/own-submissions.component.ts [hunk 1]
```diff
@@ -400,6 +401,12 @@ export class OwnSubmissionsComponent implements OnChanges, OnInit, OnDestroy {
               formID: document.formID,
               form: form.title || document.formID,
               id: document.id,
+              publicityRestrictions:
+                document.publicityRestrictions === Document.PublicityRestrictionsEnum.publicityRestrictionsPublic ?
+                  this.translate.instant('haseka.submissions.publicityRestrictions.public') :
+                    document.publicityRestrictions === Document.PublicityRestrictionsEnum.publicityRestrictionsPrivate ?
+                      this.translate.instant('haseka.submissions.publicityRestrictions.private') :
+                        this.translate.instant('haseka.submissions.publicityRestrictions.protected'),
```
Comment (olzraiti) [inline projects/laji/src/app/shared-modules/own-submissions/own-submissions.component.ts:409]: I find this a bit unreadable. I'd prefer:

```
publicityRestrictions:
	document.publicityRestrictions === Document.PublicityRestrictionsEnum.publicityRestrictionsPublic
		? this.translate.instant('haseka.submissions.publicityRestrictions.public')
		: document.publicityRestrictions === Document.PublicityRestrictionsEnum.publicityRestrictionsPrivate
			? this.translate.instant('haseka.submissions.publicityRestrictions.private')
			: this.translate.instant('haseka.submissions.publicityRestrictions.protected'),
```

Formatting it this way I find it more clear what is the "question" and what are the outcomes of that question. You can understand nested ternary operations by following the indentation.

(Maybe we should enable a linting rule for this - the way I prefer seems to be called "offset ternary expression" in https://eslint.org/docs/latest/rules/indent)

**Review by olzraiti — CHANGES_REQUESTED** [review]

File: projects/laji/src/app/shared-modules/own-submissions/own-submissions.component.ts (hunk 1 above)
Comment (rpulkka) [reply projects/laji/src/app/shared-modules/own-submissions/own-submissions.component.ts:409]: Seems reasonable. One question: Is two spaces indentation enough, or did you include the extra spaces on purpose to the example?

Comment (olzraiti) [reply projects/laji/src/app/shared-modules/own-submissions/own-submissions.component.ts:409]: In the example I used tabs, which Github apparently display with 8char width. Two spaces is enough in laji-repo :)

Comment (rpulkka) [reply projects/laji/src/app/shared-modules/own-submissions/own-submissions.component.ts]: Alright, fixed now :)

**Review by olzraiti — APPROVED** [review]
Thanks!

**Pull request merged**